src/SurveyService.Business/Commands/Question/GetQuestionCommand.cs

```diff
@@ -0,0 +1,84 @@
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.SurveyService.Broker.Requests.Interfaces;
+using LT.DigitalOffice.SurveyService.Business.Commands.Question.interfaces;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.SurveyService.Mappers.Responses.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using LT.DigitalOffice.SurveyService.Models.Dto.Models;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question.Filters;
+using LT.DigitalOffice.SurveyService.Models.Dto.Responses.Question;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Business.Commands.Question;
+
+public class GetQuestionCommand : IGetQuestionCommand
+{
+  private readonly IQuestionRepository _repository;
+  private readonly IQuestionResponseMapper _questionResponseMapper;
+  private readonly IResponseCreator _responseCreator;
+  private readonly IUserService _userService;
+  private readonly IOptionInfoMapper _optionInfoMapper;
+  private readonly IUserInfoMapper _userInfoMapper;
+  private readonly IUserAnswerInfoMapper _userAnswerInfoMapper;
+
+  public GetQuestionCommand(
+    IQuestionRepository questionRepository, 
+    IQuestionResponseMapper questionResponseMapper,
+    IResponseCreator responseCreator,
+    IUserService userService,
+    IOptionInfoMapper optionInfoMapper,
+    IUserInfoMapper userInfoMapper,
+    IUserAnswerInfoMapper userAnswerInfoMapper)
+  {
+    _repository = questionRepository;
+    _questionResponseMapper = questionResponseMapper;
+    _responseCreator = responseCreator;
+    _userService = userService;
+    _optionInfoMapper = optionInfoMapper;
+    _userInfoMapper = userInfoMapper;
+    _userAnswerInfoMapper = userAnswerInfoMapper;
+  }
+  
+  public async Task<OperationResultResponse<QuestionResponse>> ExecuteAsync(GetQuestionFilter filter)
+  {
+    OperationResultResponse<QuestionResponse> response = new();
+
+    if (filter is null || filter.QuestionId is null)
```

Comment:
QuestionId is required, you don't have to check it

Reply:
Done